crates/transaction-emitter-lib/src/args.rs

```diff
@@ -54,16 +54,19 @@ pub struct ClusterArgs {
     pub mint_args: MintArgs,
 }
 
-#[derive(Debug, Clone, Copy, ArgEnum, Deserialize, Parser, Serialize)]
+#[derive(Debug, Clone, Copy, ArgEnum, Deserialize, Parser, Serialize, PartialEq, Eq, Hash)]
 pub enum TransactionType {
+    // Deprecate P2P name
```

Comment:
n00b Q: Why do you need to deprecate instead of just deleting it?

Reply:
I just need to confirm that when we use txn emitter through cli, we replace with new name